src/ot_orchestration/dags/config/unified_pipeline.py

```diff
@@ -25,54 +25,73 @@ class PlatformConfig:
     """
 
     def __init__(self) -> None:
-        self.platform_config_path = Path(__file__).parent / "unified_pipeline.yaml"
+        self.config_path = Path(__file__).parent / "unified_pipeline.yaml"
         self.pis_config_local_path = Path(__file__).parent / "pis.yaml"
         self.etl_config_local_path = Path(__file__).parent / "etl.conf"
+        self.gentropy_config_local_path = Path(__file__).parent / "gentropy.yaml"
 
         # These are hardcoded config values that are not meant to change often.
         # It is safe to extract them from here into the config file in case they
         # become more dynamic. Just add them to self in the initialization so they
         # are available.
 
         # fmt: off
-        # The disk size for PIS vms, in GB.
-        # Note: although not all steps need this much space, it is easier to have a
-        # single value for all steps, and the machines are so short-lived that it
-        # doesn't matter much with respect to cost.
-        self.pis_disk_size = 150
-
         # The service account and scopes to use (only used by PIS so far).
         # The drive scope is needed to download spreadsheets from Google Drive
         # for the PIS otar step.
         self.service_account = "platform-input-support@open-targets-eu-dev.iam.gserviceaccount.com" #fmt: skip
         self.service_account_scopes = ["https://www.googleapis.com/auth/drive"]
 
         # Pipeline settings.
-        settings = read_yaml_config(self.platform_config_path)
-        self.gcs_url = settings["gcs_url"]
+        settings = read_yaml_config(self.config_path)
+        self.gcs_url:str = settings["gcs_url"]
+        self.release = self.gcs_url.split("/")[-1]
         self.chembl_version = settings["chembl_version"]
         self.efo_version = settings["efo_version"]
         self.ensembl_version = settings["ensembl_version"]
         self.is_ppp = settings["is_ppp"]
+        self.steps = settings["steps"]
+        self.ppp_steps = [s for s, d in self.steps.items() if d and d.get('ppp_only', False)]
 
         # PIS-specific settings.
-        self.pis_config = self.init_pis_config()
         pis_version = settings["pis_version"]
+        self.pis_config = self.init_pis_config()
         # The base image for PIS, the version tag will be appended from the config file.
         pis_image_base = "europe-west1-docker.pkg.dev/open-targets-eu-dev/platform-input-support-test/platform-input-support-test"
```

Comment:
It does not make sense to have a setting for the base image of PIS. This will never change (except when we actually move the new PIS on top of the old one. It is not something we will have to change for newer releases.

The logic here is: this class is also part of the configuration, but we only expose into actual yaml files what should be changed.

Reply:
It is up to you, it just feels wrong to hardcode again the project name (since you alredy had issues with this in the code comming from gentropy)